proto/internalapi/central/sensor_events.proto

```diff
@@ -63,7 +63,9 @@ message SensorEvent {
         uint64 sensor_hash                       = 26;
     }
 
-    message ResourcesSynced {}
+    message ResourcesSynced {
+        repeated string unchanged_ids = 1;
```

Comment:
My only concern is about the size of this message

Reply:
See comment above about problem doc and the new task to implement classic reconciliation fallback.